dms/models/directory.py

```diff
@@ -475,6 +475,14 @@ def _compute_directory_type(self):
                 record.parent_id = None
```

Comment:
In this code branch, there's `root_storage_id` assignation. For what is this field useful? Shouldn't it be the storage of the folder?

Reply:
All directories have `parent_id` except if directory is root (`is_root_directory`).
`storage_id` field is compute writable according to 
```
@api.depends("root_storage_id", "parent_id")
    def _compute_storage(self):
        for record in self:
            if record.is_root_directory:
                record.storage_id = record.root_storage_id
            else:
                record.storage_id = record.parent_id.storage_id
```
and it's reason to set None to `root_storage_id` field (show in form only if `is_root_directory` is checked)